lib/style.scss

```diff
@@ -2,6 +2,15 @@
 @use '@carbon/react/scss/zone';
 @use '@carbon/react/scss/reset';
 
+:root {
+  --focus-outline-color: hsl(205, 100%, 40%);
+}
+
+.cds--g10 {
```

Comment:
This seems wrong here, too. Do we want to _theme_ Carbon here? If so, let's instead do it in the frame (Desktop Modeler or web modeler) - https://github.com/camunda/camunda-modeler/issues/4511.

Reply:
I agree with handling Carbon theming separately https://github.com/camunda/camunda-modeler/issues/4511, but I’d argue why it shouldn’t be part of the component library. If we’re developing a component to integrate within our application, it’s important to ensure that the component is visually aligned with the application and comes with our standard styling by default. This would provide a consistent out-of-the-box experience. 

Let me know if you think otherwise

Reply:
> This would provide a consistent out-of-the-box experience.

Agreed, but the mechanism to do it for carbon-based components will be slightly different:

* We want to inject carbon as a "foundation", available to all libraries
* We want to customize it ("theme it") also as a "foundation", so carbon-based libraries don't need to be customized ("themed") locally.

Cf. https://github.com/camunda/camunda-modeler/issues/4511#issuecomment-2490218876 and following.